public/japanese/manifest.json

```diff
@@ -31,6 +31,11 @@
       "sizes": "152x152",
       "type": "image/png"
     },
+    {
+      "src": "https://news.files.bbci.co.uk/include/articles/public/japanese/images/icons/icon-180x180.png",
```

Comment:
`public/japanese/images/icons/icon-180x180.png` is actually 246x246

Reply:
Good spot

Reply:
Should be sorted now, thanks